eldev.el

```diff
@@ -4907,10 +4907,13 @@ possible to build arbitrary targets this way."
                  files-to-tar)
             (condition-case nil
                 (make-symbolic-link eldev-project-dir (expand-file-name name-version working-dir))
-              (file-error (error "FIXME: Copy")))
+               (file-error
```

Comment:
Can you think of a way of testing this? Or will this be indirectly tested on Windows just because symbolic links are not available? If so, then it's fine like this, no need for a dedicated test.

Also, indentation seems broken.

Reply:
The plan is to indirectly test this on MS-Windows.

Corrected indentation.